src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const headlines = document.querySelectorAll('li');
+
+const createSpan = () => {
+  return document.createElement('span');
+};
+
+[...headlines].map(headline => {
+  if (!headline.children.length) {
+    return;
+  }
+
+  headline.prepend(createSpan());
+
+  const span = headline.firstChild;
+
+  span.prepend(headline.firstChild.nextSibling);
+
+  span.addEventListener('click', (e) => {
+    const hidden = span.nextSibling.style;
+
+    if (hidden.display === 'none') {
+      hidden.display = '';
+
+      return;
+    };
+
+    hidden.display = 'none';
```

Comment:
We can do the same a little more easily
```suggestion
  const items = e.target.closest('span').nextSibling;

  items.hidden = !items.hidden;
```

Reply:
Hi, I see you haven't fixed previous comment, maybe you did not push your changes?
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂